test/valid-data.test.ts

```diff
@@ -53,68 +53,67 @@ function assertSchema(name: string, type: string, only: boolean = false): void {
 describe("valid-data", () => {
     // TODO: generics recursive
 
-    assertSchema("simple-object", "SimpleObject");
-
-    assertSchema("interface-single", "MyObject");
-    assertSchema("interface-multi", "MyObject");
-    assertSchema("interface-recursion", "MyObject");
-    assertSchema("interface-extra-props", "MyObject");
-
-    assertSchema("structure-private", "MyObject");
-    assertSchema("structure-anonymous", "MyObject");
-    assertSchema("structure-recursion", "MyObject");
-    assertSchema("structure-extra-props", "MyObject");
-
-    assertSchema("enums-string", "Enum");
-    assertSchema("enums-number", "Enum");
-    assertSchema("enums-initialized", "Enum");
-    assertSchema("enums-compute", "Enum");
-    assertSchema("enums-mixed", "Enum");
-    assertSchema("enums-member", "MyObject");
-
-    assertSchema("string-literals", "MyObject");
-    assertSchema("string-literals-inline", "MyObject");
-    assertSchema("string-literals-null", "MyObject");
-
-    assertSchema("namespace-deep-1", "RootNamespace.Def");
-    assertSchema("namespace-deep-2", "RootNamespace.SubNamespace.HelperA");
-    assertSchema("namespace-deep-3", "RootNamespace.SubNamespace.HelperB");
-
-    assertSchema("import-simple", "MyObject");
-    assertSchema("import-exposed", "MyObject");
-    assertSchema("import-anonymous", "MyObject");
-
-    assertSchema("type-aliases-primitive", "MyString");
-    assertSchema("type-aliases-object", "MyAlias");
-    assertSchema("type-aliases-mixed", "MyObject");
-    assertSchema("type-aliases-union", "MyUnion");
-    assertSchema("type-aliases-tuple", "MyTuple");
-    assertSchema("type-aliases-anonymous", "MyObject");
-    assertSchema("type-aliases-local-namespace", "MyObject");
-    assertSchema("type-aliases-recursive-anonymous", "MyAlias");
-    assertSchema("type-aliases-recursive-export", "MyObject");
-
-    assertSchema("type-maps", "MyObject");
-    assertSchema("type-primitives", "MyObject");
-    assertSchema("type-union", "TypeUnion");
-    assertSchema("type-union-tagged", "Shape");
-    assertSchema("type-intersection", "MyObject");
-    assertSchema("type-intersection-additional-props", "MyObject");
-
-    assertSchema("type-typeof", "MyType");
-    assertSchema("type-typeof-value", "MyType");
-    assertSchema("type-indexed-access", "MyType");
-    assertSchema("type-keyof", "MyType");
-    assertSchema("type-mapped", "MyObject");
-    assertSchema("type-mapped-single", "MyObject");
-
-    assertSchema("generic-simple", "MyObject");
-    assertSchema("generic-arrays", "MyObject");
-    assertSchema("generic-multiple", "MyObject");
-    assertSchema("generic-multiargs", "MyObject");
-    assertSchema("generic-anonymous", "MyObject");
-    assertSchema("generic-recursive", "MyObject");
-    assertSchema("generic-hell", "MyObject");
-
-    assertSchema("nullable-null", "MyObject");
+        assertSchema("simple-object", "SimpleObject");
```

Comment:
What changed here?

Reply:
looks like some formatting, probably a result of tslint or vscode formatter